examples/lib/defaultHtmlStepUi.js

```diff
@@ -102,9 +102,15 @@ function DefaultHtmlStepUi(_sequencer, options) {
           }
           html += '</select>';
         }
+        else if (inputDesc.type.toLowerCase() == 'button') {
+          
+          
+          html += '<br><button class="save btn btn-primary btn-lg" name="' + paramName + '">';
```

Comment:
We would also want to display it. The value.

Reply:
Done